backend/src/database/models/quiz.ts

```diff
@@ -0,0 +1,45 @@
+import { model, Schema, Types } from 'mongoose';
+
+const answerSchema = new Schema({
+  text: {
+    type: String,
+    required: true,
+  },
+});
+
+const questionSchema = new Schema({
+  text: {
+    type: String,
+    required: true,
+  },
+  answers: {
+    type: [answerSchema],
+    required: true,
+  },
+  points: {
+    type: Number,
+    required: true,
+  },
```

Comment:
Ta liczba mnoga wskazuje mi jakby to była relacja - może lepsze byłoby coś w stylu pointsNumber?

Reply:
mogę zmienić

Reply:
ale narazie zostawiam